rero_ils/modules/libraries/statistics.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Statistics methods about Library."""
+from elasticsearch_dsl import A
+
+from rero_ils.modules.documents.api import DocumentsSearch
+
+
+def get_document_type_repartition(library, document_type=None, excludes=None):
+    """Get document type repartition about a library.
```

Comment:
Suggestion: "Get document type repartition about a library." -> "Get the number document type grouped by library."